src/components/SearchPage/SearchPageComponent.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+import { useDebouncedCallback } from 'use-debounce';
+import { useDispatch, useSelector } from 'react-redux';
+import { searchQueryChangeAction } from '../SearchPage/SearchPageAction';
+
+export function SearchPageComponent() {
+  const results = useSelector(state => state.notesMindMap.data);
```

Comment:
Why are the search results stored in `state.notesMindMap.data` ?
Sounds quite confusing to me. Shouldn't it be something like `state.searchResults` ?

Reply:
I create a SearchPage Sagas.js and add a new case to SearchPageReducer.js to solve that confusion.

Reply:
Great idea! Well done!